databases/week2/Creating a database.sql

```diff
@@ -0,0 +1,38 @@
+-- creating database --
+
+CREATE DATABASE Cake_Bakery;
+
+USE Cake_Bakery;
+
+SET NAMES utf8mb4;
+
+  
+CREATE TABLE `bakery_offices` (
+  `offices_id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
simple `id` would do

Reply:
Is this only about the name :) ? Cant we just put anything as a column name ? In this case I have created it as 'offices_id' as I wanted to make sure that column name is understandable and not ambiguous with the other primary keys that we have in the rest of the tables.

Reply:
It is not just about the naming.
If you have fx a book, then it has a name. It has not a book_name, it has an id and not a book_id.
When you use this id as foreign key then you call it book_id, so when you see book_id you realize that it is a foreign key.
Another example is this:
if you are coding then you might create a class `Book `and an object of the class `book`
what do you prefer to write? `book.book_id` or `book.id`